service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -4,7 +4,7 @@
     C110A("FL-PLW-APP-ENG-00095.doc", "c110a_application_%s.pdf"),
     C6("FL-PLW-HNO-ENG-00092.doc", "Notice_of_proceedings_c6.pdf"),
     C6A("FL-PLW-HNO-ENG-00093.doc", "Notice_of_proceedings_c6a.pdf"),
-    SDO("FL-PLW-STD-ENG-00099.doc", "standard_directions_order.pdf"),
+    SDO("FL-PLW-STD-ENG-00099.doc", "standard-directions-order.pdf"),
```

Comment:
is there any reason why we name this pdf with '-' rather than '_' like that others

Reply:
Hardcoded values where removed from the status enum. Originally using 'standard-direction-order.pdf'. It looks like the format was changed to keep tests passing rather than updating tests.